setup.py

```diff
@@ -30,7 +30,11 @@
 
 
 LIBPG_QUERY_DIR = str(here / 'libpg_query')
+INCLUDE_DIR = str(here / 'libpg_query' / 'tmp' / 'postgres' / 'src' /
+                  'include')
 
+CFLAGS="-flto -fvisibility=hidden -fdata-sections -ffunction-sections"
+LDFLAGS="-Wl,--gc-sections,--as-needed"
```

Comment:
@PierreDucroquet, @rdunklau: shouldn't these be passed to the `Extension` below as respectively `extra_compile_args` and `extra_link_args`?

Reply:
You're right, except we need to pass all the flags as compile and link args. I pushed the fix.